packages/go/constants/constants.go

```diff
@@ -0,0 +1,78 @@
+package constants
+
+type Ecosystem string
+
+const (
+	EcosystemGo            Ecosystem = "Go"
```

Comment:
nit: can we sort this alphabetically to make it easier to parse as a human?

Reply:
yup I can do that manually, though just noting the plan is to have this automatically generated which'll take care of that too

Reply:
done - I've not sorted the others as they felt more like they'd been arranged in a natural order, but let me know if you think they should be sorted too